components/brave_rewards/core/logging/logging_util_unittest.cc

```diff
@@ -10,6 +10,8 @@
 
 namespace brave_rewards::internal {
 
+bool ShouldLogHeader(const std::string& header);
+
```

Comment:
It feels weird that the unit test needs to provide the declaration for the utility function. Or am I missing something here?

Reply:
`ShouldLogHeader` is internal to `UrlRequestToString`, so it shouldn't be exposed via `logging_util.h`, but there's already some unit tests for it, so I can't put it behind an anonymous namespace in `logging_util.cc`. This still does not make it translation-unit-local (as anybody can declare the function just like the test does), but slightly better than making it part of the API, imo.

Reply:
I think we should probably just keep it as part of the API if unit tests need it, and until they don't, since it's less for a maintainer to figure out.

Reply:
Yup, that works, too, if you guys prefer that! 👍🏻